app/views/categories/index.html.erb

```diff
@@ -0,0 +1,36 @@
+<main>
+<div class="container new-item d-flex flex-column align-items-center px-0">
+    <div class="ui-header d-flex justify-content-between align-items-center">
+       <i class="fa-solid fa-bars"></i>
+        <h2 class="ui-heading">Categories</h2>
+        <a href="/logout" data-method="delete">
+            <i class="fa-solid fa-sign-out"></i>
+        </a>
+    </div>
+
+    <div class="categories col-12 p-2 grow-1">
+        <% if @categories.empty? %>
+      <div class="empty-message">You have not added any Category. Add one by clicking the button below.</div>
+        <% else %> <% @categories.each do |category| %>
+
+        <a href="<%= category_items_path(category.id) %>">
+
+            <div class="single-category col-12 d-flex justify-content-between align-items-center">
+                <div class="d-flex justify-content-start align-items-center left">
+                <img src="<%= category.icon %>" alt="icon" class="category-icon">
+                <p class="category-name"><%= category.name %></p>
+                </div>
+                <p class="category-total justify-self-"> $ <%= category.total_transaction_amount %> </p>
+            </div>
```

Comment:
- [ ] nice design but there is a date missing in the UI and the price should be bold also try and keep similar font for the name and price too :+1: 
given design :point_down: 
![Screenshot from 2023-05-06 13-54-49](https://user-images.githubusercontent.com/66097371/236625671-c32fe8eb-12be-4e58-ae75-a0d218838327.png)
your design :point_down: 
![Screenshot from 2023-05-06 14-00-34](https://user-images.githubusercontent.com/66097371/236625809-394d97c0-741a-4961-b160-a29d450da11f.png)

Reply:
hi @VICTOR-CHUKWU, 

1. There is no explicit instruction for showing the date in the requirements. 
2. As the font in the design is not free, I used a similar font from google fonts.